src/Blankslate/Blankslate.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import styled from 'styled-components'
+import Box from '../Box'
+import {Button} from '../Button'
+import {get} from '../constants'
+import Link from '../Link'
+
+const StyledBlankslate = styled.div`
+  display: grid;
+  justify-items: center;
+  padding: ${get('space.5')};
+
+  &[data-border='true'] {
+    border: ${get('borderWidths.1')} solid ${get('colors.border.default')};
+    border-radius: ${get('radii.2')};
+  }
+
+  &[data-narrow='true'] {
+    margin: 0 auto;
+    max-width: 485px;
+  }
+
+  &[data-spacious='true'] {
+    padding: ${get('space.9')} ${get('space.6')};
+  }
+
+  .BlankSlateAction {
+    margin-top: ${get('space.3')};
+  }
+
+  .BlankSlateAction:first-of-type {
+    margin-top: ${get('space.4')};
+  }
+
+  .BlankSlateAction:last-of-type {
+    margin-bottom: ${get('space.2')};
+  }
+`
+
+export type BlankslateProps = React.PropsWithChildren<{
+  /**
+   * Add a border around this component
+   */
+  border?: boolean
+
+  /**
+   * Constrain the maximum width of this component
+   */
+  narrow?: boolean
```

Comment:
I feel like this should just be a variant with `narrow` and `spacious` but for now it makes sense to just follow the existing API!